packages/host/src/AppDrawer.tsx

```diff
@@ -21,7 +21,11 @@ import {
 } from '@openmsupply-client/common';
 import clsx from 'clsx';
 
-const CustomersNav = React.lazy(() => import('customers/Nav'));
+const CustomersNav = React.lazy(() =>
+  process.env['NODE_ENV'] === 'test'
+    ? import('../../customers/src/Nav')
+    : import('customers/Nav')
```

Comment:
Hm. This seemed easiest but is probably not the best way to go..

Reply:
what about lifting it out to a helper method. still just as clumsy, but the code looks nicer! ( and if we think of another way to make it work, then easy to change )
```
const lazyLoad = (path: string) => React.lazy(() =>
  process.env['NODE_ENV'] === 'test'
    ? import(`../../${path}`)
    : import(path);

const CustomersNav = lazyLoad('customers/Nav');
```

Reply:
Nice. I change this a little in my next PR so will address it soon!